### PR TITLE
Some edits on "Design of Tokenscript"

### PR DESCRIPTION
**Pull request by theBergmann**
Made some edits.  Not sure about this chapter - ZW wants to rewrite it - but I started with trying to structure it. Hope it help.s

### CODE REVIEW

File: doc/design_paper.md [hunk 1]
```diff
@@ -811,55 +819,47 @@ We demonstrate the portion of Tokenscript related to messaging.
...
+The categorisation isn't capturing the full spectrum of the tokens we could and may overlap in some cases. Taking the 1% per cent property token we demonstrated earlier as an example, each of such token is fungible with another issued by the same issuer for the same property. At the same time, someone could create a property token with 0.88% to attract Chinese investors, because 8 is a lucky number in China. These token would not be fungible with the 1% token. Onviously, a percentage of ownership of property A and a percentage of ownership of property B are not fungible with each other. A mix-token with 1% or A and B would be fungible, but at the same time, you could build individualized, non-fungible tokens which represent an investor's own pick of a property mix. The design of Tokenscript has to reflect that the line between fungible and non-fungible tokens is not perfect and that there might be mixed forms.
```
Comment (James-Sangalli) [inline doc/design_paper.md]: These token -> this token. Onviously - Obviously

File: doc/design_paper.md [hunk 2]
```diff
@@ -811,55 +819,47 @@ We demonstrate the portion of Tokenscript related to messaging.
...
 
-This paper re-introduces the concept of attestations - it has been there for decades but wasn't fully utilized. From there, this paper categorises tokens as "blockchain token" and "attestation". The former type includes both fungible and non-fungible tokens. The latter type "attestation" will be explained here.
+Further, this paper re-introduces the concept of attestations - it has been there for decades but wasn't fully utilized. From there, this paper categorises tokens as "blockchain token" and "attestation". The former type includes both fungible and non-fungible tokens. The latter type "attestation" will be explained here.
```
Comment (James-Sangalli) [inline doc/design_paper.md]: Blockchain tokens and attestations sounds more natural

File: doc/design_paper.md [hunk 3]
```diff
@@ -811,55 +819,47 @@ We demonstrate the portion of Tokenscript related to messaging.
...
 
-To explain the use case where the *issuing* of attestation has to happen on the blockchain or with blockchain trace, take the aeroplane engine for example, with a substantial resale value, the repair facts of this engine, in the form of attestations, affects valuation significantly. Such attestations are in the seller's wallet, but an aeroplane service provider must add a hash of such an attestation each time the engine undergoes maintenance. The buyers would not purchase if they are not presented with these attestations that match the blockchain records.
+To explain the use case where the *issuing* of attestation has to happen on the blockchain or with blockchain trace, take the example of an aeroplane engine. It is represented by a token and carries a lot of attestation information, like the repair facts, which significantly affets its valuation (and security and insurance properties). Such attestations are in the seller's wallet, but an aeroplane service provider must add a hash of such an attestation each time the engine undergoes maintenance. The buyers would not purchase it if they are not presented with these attestations that match the blockchain records.
```
Comment (James-Sangalli) [inline doc/design_paper.md]: affets -> Affects

File: doc/design_paper.md [hunk 4]
```diff
@@ -811,55 +819,47 @@ We demonstrate the portion of Tokenscript related to messaging.
...
+To explain the use case when the *revocation* of an attestation has to happen on the blockchain, let's consider an attestation called FIFA ticket. Issued by the event's organiser, it attests the owner's right to enter the venue, usually after the user has paid or was gifted the ticket. Let's assume 90% of the tickets are purchased with non-crypto currency, therefore these tickets would not have a trace on the blockchain. However, if a ticket's owner decides to sell his tickets on the blockchain following the corresponding smart contract rules, the ticket has to be used as the input of such a transaction and considered consumed, while a blockchain token representing the same entitlement would be created and traded. The writes of this paper organised a FIFA ticket experiment in mid-2018 to test the concepts, and internally we call such an attestation "a spawnable" as its use spawns a blockchain token. The detail of that experiment can be found in another paper [cite].
```
Comment (James-Sangalli) [inline doc/design_paper.md]: writes -> writers

**Comment by theBergmann** [comment]
Sorry for the late reply. Updated the file according to your comments. I hope I didn't miss a comment.

**Comment by James-Sangalli** [comment]
Thanks!

**Pull request merged**